docs/source/concept_models/vivarium_swissre_stomachcancer/concept_model.rst

```diff
@@ -441,8 +441,12 @@ We first need to obtain an atrophy state. To do that we give every simulant an a
 
 With the simulant's sex, age and atrophy propensity, use the tables above to figure out what atrophic state the propensity corresponds to and assign this to the simulant. If the propensity is < the proportion in the table, they are atrophic+. Update the simulant's atrophic state as they age through the simulation.   
 
+:underline:`B. Determine H. pylori and atrophy for simulants initalized into the pre-clinical state`
 
-:underline:`B. Obtain H. pylori status conditional upon age and atrophic state`
+  Assign all simulants initialized in the pre-clinical cancer state to atrophy positive.
```

Comment:
This seems complicated, can I not do it, and instead we include as a limitation that we have not correlated the preclinical stomach cancer status and h. pylori/atrophy status?

Reply:
ok- I've gotten rid of this and added as a limitation.